helm/observability-operator/files/alertmanager/alertmanager.yaml.helm-template

```diff
@@ -0,0 +1,504 @@
+global:
+  resolve_timeout: 5m
+{{- if .Values.alerting.proxyURL }}
+  http_config:
+    proxy_url:  {{ .Values.alerting.proxyURL }}
```

Comment:
I think this is not useful anymore with the env variables automatically set on all pods. Do you think you could test without it on goat? Because I'm not sure we can get this value anyway in config